src/lib/RequestParser.php

```diff
@@ -8,6 +8,9 @@
 
 /**
  * Interface for Request parsers.
+ *
+ * @deprecated 4.6.11
+ * use \Ibexa\Contracts\Rest\UriParser\UriParserInterface and \Symfony\Component\Routing\RouterInterface instead
```

Comment:
I think it should be in one line and match our deprecation document:

> #VERSION# The "#CLASSNAME#" class is deprecated, will be removed in #VERSION#. Use "#CLASSNAME#" instead.

See https://ibexa.atlassian.net/wiki/spaces/ENG/pages/5799937/PHP+Conventions+Deprecation+standards